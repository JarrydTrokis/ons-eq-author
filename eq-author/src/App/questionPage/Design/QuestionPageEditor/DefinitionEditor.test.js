import React from "react";
import { shallow } from "enzyme";

import DefinitionEditor from "./DefinitionEditor";

const createWrapper = props => shallow(<DefinitionEditor {...props} />);

describe("DefinitionEditor", () => {
  let wrapper, props;

  beforeEach(() => {
    props = {
      label: "foobar",
      children: <p>Test</p>
    };

    wrapper = createWrapper(props);
  });

  it("should render", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
