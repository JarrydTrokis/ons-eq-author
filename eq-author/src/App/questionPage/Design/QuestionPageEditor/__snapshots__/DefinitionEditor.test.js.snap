// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DefinitionEditor should render 1`] = `
<Fragment>
  <DefinitionEditor__DefinitionLabel>
    foobar
  </DefinitionEditor__DefinitionLabel>
  <DefinitionEditor__Definition>
    <DefinitionEditor__Padding>
      <p>
        Test
      </p>
    </DefinitionEditor__Padding>
  </DefinitionEditor__Definition>
</Fragment>
`;
